package.json

```diff
@@ -6,6 +6,10 @@
   "scripts": {
     "test": "lerna run test",
     "test:coverage": "lerna run test --stream -- --coverage",
+    "cy:run:api": "cypress run --config integrationFolder=e2e/cypress/api,baseUrl=http://localhost:7777",
```

Comment:
let' remove api tests from cypress completely because  they are covered in integration test suite